src/commands/image/imageSource/buildImageInAzure/ImageNameStep.ts

```diff
@@ -35,19 +41,35 @@ export class ImageNameStep extends AzureWizardPromptStep<BuildImageInAzureImageS
 
         return undefined;
     }
+
+    static generateSuggestedImageName(repositoryName: string = 'hello-world'): string {
```

Comment:
I'd say remove the default name here. I'm pretty sure everywhere you use it has a string passed into it. And even if not, I think I'd prefer we provide a default name that makes sense in context rather than "hello-world"

Reply:
This was really a just-in-case suggested name fallback because `getSuggestedRepositoryName` can return `undefined`.  My hunch is that by the time it gets to the end, it probably is always a string, but I wasn't 100% confident and I didn't write that section, so just to be safe I added the default.

Reply:
Actually I have a better idea. Coming soon to a theater near you ™️